UPGRADE-3.4.md

```diff
@@ -68,6 +68,9 @@ Process
 
  * The `Symfony\Component\Process\ProcessBuilder` class has been deprecated,
    use the `Symfony\Component\Process\Process` class directly instead.
+   
+ * The `Symfony\Component\Process\Process` __construct `cwd` argument will throw 
+   a deprecation warning on `run` if the folder does not exist.
```

Comment:
Should I put that in the upgrade log before the exception is even implemented? Or should there be a separate PR for the exception?